src/gui/wallet.go

```diff
@@ -499,7 +483,8 @@ func walletNewAddresses(gateway *daemon.Gateway) http.HandlerFunc {
 		}
 
 		if err := Wg.SaveWallet(wltID); err != nil {
-			wh.Error500(w, "")
+			logger.Error("save wallet failed when generate new addresses: %v", err)
```

Comment:
There are multiple places this change is needed in this file, I am referencing this documentation: https://godoc.org/github.com/op/go-logging#Logger.Error